wolfcrypt/src/ecc.c

```diff
@@ -9148,8 +9148,8 @@ int wc_ecc_import_raw_ex(ecc_key* key, const char* qx, const char* qy,
 }
 
 /* Import x, y and optional private (d) as unsigned binary */
-int wc_ecc_import_unsigned(ecc_key* key, byte* qx, byte* qy,
-                   byte* d, int curve_id)
+int wc_ecc_import_unsigned(ecc_key* key, const byte* qx, const byte* qy,
+                   const byte* d, int curve_id)
```

Comment:
This will break compatibility for a public API. I agree with the change, but is there another way?

Reply:
Going from not-const to const is safe as existing code will work.
This was an attempt to stop callers from having to cast const pointers.